reporter-web-app/build.gradle

```diff
@@ -0,0 +1,10 @@
+plugins {
+    id "com.moowork.node" version "1.2.0"
+}
+
+node {
+    version = "8.11.4"
```

Comment:
Where do these versions come from?

Reply:
These are the currenty stable/LTS releases.